src/ts/Makefile

```diff
@@ -2,6 +2,8 @@
 check-style:
 	npm --prefix . run lint-ci
 
+lint:
+	npm --prefix . run lint
```

Comment:
adds `lint` Make command to allow `make frontend-lint`

Reply:
in the future we should change `check-style` to `lint-ci` so that the lint make targets are more consistent with each other.